### PR TITLE
Add GitHub actions for CDK continuous deployment 

### PR DESCRIPTION
**Pull request by junqiu-lei**
### Description
`planet-cdk-cd.yml` will be triggered by push event to update planet tiles generation CDK stack.

`test-cdk-cd.yml` will be triggered by pull request event to update test tiles generation CDK stack and run ECS test tiles generation task.
 
### Issues Resolved
https://github.com/opensearch-project/maps/issues/2 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/planet-cdk-cd.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          npm install -g aws-cdk
+          cd tiles-generation/cdk
+          npm install aws-cdk-lib
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
```
Comment (VijayanB) [inline .github/workflows/planet-cdk-cd.yml:33]: why not make this configurable?

Comment (junqiu-lei) [reply .github/workflows/planet-cdk-cd.yml:33]: sure, will update.

File: .github/workflows/test-cdk-cd.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          npm install -g aws-cdk 
+          cd tiles-generation/cdk
+          npm install aws-cdk-lib
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
```
Comment (martin-gaievski) [inline .github/workflows/test-cdk-cd.yml:34]: same comment as Vijay's, we can do it configurable

Comment (junqiu-lei) [reply .github/workflows/test-cdk-cd.yml:34]: sure, will update to make it configurable.

**Review by martin-gaievski — APPROVED** [review]
looks good, thx

**Review by Shivamdhar — APPROVED** [review]

**Comment by junqiu-lei** [comment]
Since PR triggered actions will need access to GitHub secrets, for security concern, I just added a approval workflow before PR can be able to run the actions workflow. The test here: https://github.com/junqiu-lei/maps-1/runs/5761090091?check_suite_focus=true

**Review by martin-gaievski — APPROVED** [review]

**Review by VijayanB — APPROVED** [review]

**Pull request merged**